Marlin/src/lcd/marlinui.h

```diff
@@ -653,6 +653,16 @@ class MarlinUI {
     static void move_axis_screen();
   #endif
 
+  #if HAS_WIRED_LCD && HAS_LCD_MENU && LCD_TIMEOUT_TO_STATUS > 0
+    #define IS_PAUSE defer_return_to_status
+  #else
+    #define IS_PAUSE false
+  #endif
+
+  FORCE_INLINE static bool is_pause() {
+    return IS_PAUSE;
+  }
```

Comment:
- This is rejected. The `defer_return_to_status` flag has nothing to do with `MarlinUI` or anything else being "paused."
- Although some screen handlers call `defer_status_screen` continually, it only needs to be called once to defer the status screen.
- Any call to `return_to_status` or `goto_screen` should already clear the `defer_return_to_status` flag, so there should be no need to call `defer_status_screen` after `goto_screen` or `return_to_status`.

Reply:
Hmm, evidently the `defer_return_to_status` flag is left intact (set or unset) by `goto_screen` unless it is returning to the status screen. So, once `defer_return_to_status` is set, it does remain unchanged until some other screen handler changes it.

To ensure that menus and screens behave as intended, we could add the `defer_return_to_status` flag to the screen history at the cost of 1 byte per level, restoring it whenever the "go back in history" function is invoked.